semgrep-core/parsing/Parse_javascript_tree_sitter.ml

```diff
@@ -1758,10 +1758,13 @@ let parse file =
     program env ast
   with
     (Failure "not implemented") as exn ->
-      let s = Printexc.get_backtrace () in
-      pr2 "Some constructs are not handled yet";
-      pr2 "CST was:";
-      CST.dump_tree ast;
-      pr2 "Original backtrace:";
-      pr2 s;
+      (* This debugging output is not JSON and breaks core output
```

Comment:
do we need to comment this out?

Reply:
But it is printed on stderr. It should not matter. The json is printed on stdout

Reply:
Commenting it out fixes the bug, so 🤷

Reply:
@aryx when you get back to it, you probably should check if by any chance semgrep merges semgrep-core's stderr into stdout.